src/test/java/uk/ac/ebi/eva/commons/models/converters/data/VariantToDBObjectConverterTest.java

```diff
@@ -138,8 +138,8 @@ private BasicDBObject buildMongoVariant(boolean withFiles) {
         samples.add(na002);
 
         //Setup mongoVariant
-        BasicDBObject mongoVariant = new BasicDBObject("_id", "1_1000_A_C")
-                .append(VariantDocument.IDS_FIELD, variant.getIds())
+        Document mongoVariant = new Document("_id", "1_1000_A_C")
+//                .append(VariantDocument.IDS_FIELD, variant.getIds())
```

Comment:
why is this commented?

Reply:
Thanks, I forgot to delete that line but I just did it